test/inductor/test_cutlass_backend.py

```diff
@@ -616,6 +617,28 @@ def select_no_algorithm(*args, **kwargs):
                             cuda_template_count += 1
                     assert cuda_template_count > 0, "No CUDATemplateCaller choices"
 
+    @unittest.skipIf(not SM80OrLater, "need sm_90")
+    @unittest.skipIf(config.is_fbcode(), "fbcode requires different CUTLASS path setup")
+    @unittest.mock.patch.dict(os.environ, {"PATH": _get_path_without_sccache()})
+    def test_get_max_alignment(self):
+        l4 = FixedLayout("cpu", torch.half, size=(1, 2, 4), stride=(0, 4, 1))
+        m4 = get_max_alignment(l4)
+        assert (
```

Comment:
I think python tests should use `self.assertEquals`?

Reply:
Thanks for the review. I addressed all comments.